File: framework/wcf/Extensibility/Transactions/TransactionMessagePropertyUdpTransport/VB/UdpTransport/SampleProfileUdpBindingCollectionElement.vb
'  Copyright (c) Microsoft Corporation. All rights reserved.

Imports System.ServiceModel.Configuration

Namespace Microsoft.ServiceModel.Samples
	''' <summary>
	''' Binding Section for Udp. Implements configuration for SampleProfileUdpBinding.
	''' </summary>
	Public Class SampleProfileUdpBindingCollectionElement
		Inherits StandardBindingCollectionElement(Of SampleProfileUdpBinding, SampleProfileUdpBindingConfigurationElement)
	End Class
End Namespace

File: framework/wcf/Extensibility/Transport/Udp/CS/UdpTransport/SampleProfileUdpBindingCollectionElement.cs
//----------------------------------------------------------------
// Copyright (c) Microsoft Corporation.  All rights reserved.
//----------------------------------------------------------------

using System.ServiceModel.Configuration;
using System.ServiceModel.Channels;

namespace Microsoft.Samples.Udp
{

    /// <summary>
    /// Binding Section for Udp. Implements configuration for SampleProfileUdpBinding.
    /// </summary>
    public class SampleProfileUdpBindingCollectionElement : StandardBindingCollectionElement<SampleProfileUdpBinding, SampleProfileUdpBindingConfigurationElement>
    {
    }
}
